### PR TITLE
Refactor PostCard

### PR DESCRIPTION
**Pull request by cmdea**
Issue #51 

⚠️ I noticed that the emotion components were rendering duplicate css while working on this PR and I fixed it by adding a wrapper emotion component to `_app.tsx`. This seems to fix the problem, but if you know better or have encountered this before let me know.

### CODE REVIEW

**Review by nring — COMMENTED** [review]
I don't know about `emotion` specifically, but can you remove your fix and verify that the duplicates still happen in a full build? I've seen issues with Next where hot reloading will apply duplicate styles, but it resolves itself in production.

> ⚠️ I noticed that the emotion components were rendering duplicate css while working on this PR and I fixed it by adding a wrapper emotion component to _app.tsx. This seems to fix the problem, but if you know better or have encountered this before let me know.

File: styles/screen-fixings.css [hunk 1]
```diff
@@ -6,10 +6,6 @@ html.casper body.fade-in {
   opacity: 0;
 }
 
-html.casper .next-image-wrapper {
```
[the remark this answers is not in the document]
Comment (cmdea) [reply styles/screen-fixings.css:9]: It's still used, putting it back in

File: components/PostCard/index.tsx [hunk 2]
```diff
@@ -13,235 +10,119 @@ import { GhostPostOrPage, GhostSettings } from '@lib/ghost'
...
-    isFeatured: post.featured,
-    isImage: !!featImg
+    slug,
+    url
   })
-  const isFirstPost = num !== undefined && num < 1
-  const authors = post?.authors?.filter((_, i) => (i < 2 ? true : false))
-  const textColor = isFirstPost || isColorInverted ? 'white' : 'onyx'
 
-  if (isFirstPost) {
```
[the remark this answers is not in the document]
Comment (cmdea) [reply components/PostCard/index.tsx:57]: My pleasure 😀

**Comment by cmdea** [comment]
> I don't know about `emotion` specifically, but can you remove your fix and verify that the duplicates still happen in a full build? I've seen issues with Next where hot reloading will apply duplicate styles, but it resolves itself in production.
> 
> > ⚠️ I noticed that the emotion components were rendering duplicate css while working on this PR and I fixed it by adding a wrapper emotion component to _app.tsx. This seems to fix the problem, but if you know better or have encountered this before let me know.

Yes it's also happening on PROD. Screenshot below from today. Here's [a link the the emotion issue](https://github.com/emotion-js/emotion/issues/2436) describing the behavior and the fix I used.
Screen Shot 2022-05-20 at 9 20 09 AM

**Comment by nring** [comment]
> Yes it's also happening on PROD. Screenshot below from today. Here's [a link the the emotion issue](https://github.com/emotion-js/emotion/issues/2436) describing the behavior and the fix I used. Screen Shot 2022-05-20 at 9 20 09 AM

Ahhh okay. This seems like the path of least resistance, and less costly than using `<CacheProvider>`.

**Review by nring — APPROVED** [review]
🎉

**Pull request merged**